.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .settings
 *.log
 dist/
+.idea/
```

Comment:
Such things should be handled by user's [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c#global-gitignore-contents).